docs/shared_parsers_catalog/neutron_plugin.rst

```diff
@@ -0,0 +1,9 @@
+Neutron Plugin Configuration
+============================
+
+file ``/etc/neutron/plugin.ini``
```

Comment:
@PaulWay - just like @bfahr commented in #147, would you like to change it back to the current documentation standard?